pkg/engine/virtualmachines.go

```diff
@@ -541,6 +541,12 @@ func createAgentAvailabilitySetVM(cs *api.ContainerService, profile *api.AgentPo
 		}
 	}
 
+	if to.Bool(profile.UltraSSDEnabled) {
```

Comment:
Let's add this equivalent foo to the `CreateMasterVM` func so that the master virtualMachine resources have the same `UltraSSDEnabled` configuration